src/Automation/Interfaces/IScanner.cs

```diff
@@ -21,11 +22,31 @@ public interface IScanner
         /// All exceptions are wrapped in <see cref="AxeWindowsAutomationException"/>.
         /// If the exception was not thrown by AxeWindows automation, the <see cref="Exception.InnerException"/> property
         /// will contain the exception.
+        /// Calling this function is equivalent to passing false to <see cref="IScanner.Scan(bool)"/> and passing false for <c>enableMultipleWindows</c>.
         /// </remarks>
         /// <returns>Information about the scan and any issues detected</returns>
         /// <exception cref="AxeWindowsAutomationException"/>
         ScanResults Scan();
 
+        /// <summary>
+        /// Run AxeWindows automated tests on all windows or the first window found.
+        /// </summary>
+        /// <remarks>
+        /// If a value was provided in <see cref="Config.OutputDirectory"/>,
+        /// output files may be written to the specified directory.
+        /// (Note: no output files will be written if no errors were found.)
+        /// An exception may be thrown if the value of <see cref="Config.ProcessId"/> is invalid
+        /// or if the directory provided in <see cref="Config.OutputDirectory"/> cannot be created or accessed.
+        /// All exceptions are wrapped in <see cref="AxeWindowsAutomationException"/>.
+        /// If the exception was not thrown by AxeWindows automation, the <see cref="Exception.InnerException"/> property
+        /// will contain the exception.
+        /// If <paramref name="enableMultipleWindows"/> is set to true, all windows will be scanned. If it is set to false, only the first window will be scanned.
+        /// </remarks>
+        /// <returns>Information about the scan and any issues detected</returns>
+        /// <exception cref="AxeWindowsAutomationException"/>
+        IReadOnlyCollection<ScanResults> Scan(bool enableMultipleWindows);
```

Comment:
I think we only add 1 new method to this interface:
```suggestion
        IReadOnlyCollection<ScanResults> Scan(string scanId);
```